torch/_higher_order_ops/flex_attention.py

```diff
@@ -179,11 +179,12 @@ def trace_flex_attention(
     ] + [torch.zeros((), dtype=torch.int) for _ in range(4)]
     with TransformGetItemToIndex():
         score_graph = make_fx(score_mod)(*example_vals, *other_buffers)
-    proxy_mode.tracer.root.register_module("sdpa_score", score_graph)
+    qualname = proxy_mode.tracer.get_fresh_qualname("sdpa_score")
+    proxy_mode.tracer.root.register_module(qualname, score_graph)
     node_args = (query, key, value, score_graph, *other_buffers)
     proxy_args = pytree.tree_map(proxy_mode.tracer.unwrap_proxy, node_args)
     out_proxy = proxy_mode.tracer.create_proxy(
-        "call_function", flex_attention, proxy_args, {}, name="flex_attention"
+        "call_function", flex_attention, proxy_args, {}
```

Comment:
do we loos nice names?

Reply:
No the name is still based off the hop operator.